File: snippets/rust/delete_full_snapshot.rs
async fn main() -> Result<(), qdrant_client::QdrantError> {
    use qdrant_client::Qdrant;

    let client = Qdrant::from_url("http://localhost:6334").build()?;

    client.delete_full_snapshot("{snapshot_name}").await?;

    Ok(())
}

File: snippets/go/delete_full_snapshot.go
package client

import (
	"context"

	"github.com/qdrant/go-client/qdrant"
)

func deleteFullSnapshot() {
	client, err := qdrant.NewClient(&qdrant.Config{
		Host: "localhost",
		Port: 6334,
	})
	if err != nil {
		panic(err)
	}

	client.DeleteFullSnapshot(context.Background(), "{snapshot_name}")
}
